lib/speech/operation.js

```diff
@@ -0,0 +1,94 @@
+/*!
```

Comment:
Can we add functionality here that we have inside of https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/lib/compute/operation.js, specifically, making it an EventEmitter and having a convenient way of letting a user poll for updates. Maybe the basic functionality of `module:compute/operation` can be moved to `lib/common/operation.js`, then borrowed in both places.